configure.ac

```diff
@@ -227,6 +227,20 @@ AM_COND_IF([ENABLE_MANPAGES], [
 	AS_IF([test "x$RST2MAN" = "x"], [AC_MSG_ERROR([[--enable-manpages requires rst2man]])])
 ])
 
+AC_ARG_WITH([bash-completion-dir],
```

Comment:
Ok - to be honest the configure.ac code was taken from an on-line guide I found. I will re-work it.

Reply:
After a lot of thought, I've revised the autoconf files in commit 78ec852
This now uses "enable-" not "with-" because we are choosing what to install, not what to build it with. I'm not at all sure which is best. I am happy to revise this again if you prefer "-with" or can suggest a better approach.
[I used to be a software engineer back in the '90s and hated autoconf then. Now coming back to it 25 years later and I hate it even more]

Reply:
> This now uses "enable-" not "with-" because we are choosing what to install, not what to build it with. I'm not at all sure which is best.

I would keep it as `--with` because nothing extra is enabled in the project itself - this bash completions script could be a separate project (e.g. another git repo).

> hated autoconf then. Now [...] I hate it even more

:D Yeah, it's not the best tool for configuration, but it's widely adopted. There is a PR #160 for changing autotools to meson, but since there was a comment that not everyone can use it, I'm keeping autotools as it is.